db/re/mob_db.conf

```diff
@@ -82288,15 +82288,420 @@ mob_db: (
 //3007,EP14_MORS_DUMMY
 //3008,EP14_MORS_HIDDEN
 //3009,EP14_3_DEATH_BOSS
-//3010,EP14_3_DEATH_A_MOB1
-//3011,EP14_3_DEATH_A_MOB2
-//3012,EP14_3_DEATH_A_MOB3
-//3013,EP14_3_DEATH_B_MOB1
-//3014,EP14_3_DEATH_B_MOB2
-//3015,EP14_3_DEATH_B_MOB3
-//3016,EP14_3_DEATH_C_MOB1
-//3017,EP14_3_DEATH_C_MOB2
-//3018,EP14_3_DEATH_C_MOB3
+{
+	Id: 3010
+	SpriteName: "EP14_3_DEATH_A_MOB1"
+	Name: "Corrupt Orc Baby"
+	Lv: 158
+	Hp: 250000
+	Sp: 1
+	Exp: 12390
+	JExp: 16104
+	AttackRange: 1
+	Attack: [1036, 1415]
+	Def: 240
+	Mdef: 50
+	Stats: {
+		Str: 120
+		Agi: 85
+		Vit: 80
+		Int: 60
+		Dex: 88
+		Luk: 50
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Small"
+	Race: "RC_DemiHuman"
+	Element: ("Ele_Earth", 2)
+	Mode: {
+		CanMove: true
```

Comment:
according to rA, this monster has AI = 4, which if I understood right, it should be:

04: 0x3885 (angry, change-target melee/chase)

that bitmask actually adds a few others: CanMove (1), Aggresive (4), CanAttack (128)

is the Mode from this PR correct?

Reply:
Ok, I'll use the AI that rAthena has. I didn't find any other information regarding this